docs/en/Running-Alluxio-Locally.md

```diff
@@ -73,20 +73,24 @@ By default, Alluxio filesystem uses [RAMFS](https://www.kernel.org/doc/Documenta
 
 ## Can I still try Alluxio on Linux without sudo privileges?
 
-If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted by the system admin and accessible for read/write-operations by the user. In this case you can specify the path in
+If you have no sudo privileges on Linux, for Alluxio Filesystem to work, it requires a RAMFS (e.g., `/path/to/ramdisk`) already mounted
+by the system admin and accessible for read/write-operations by the user. Alternatively, you can also specify Linux [tmpFS](https://en.wikipedia.org/wiki/Tmpfs)  
+as the data storage. Tmpfs is a temporary file storage backed by memory (e.g., typically `/dev/shm` on Linux), but may use SWAP space and
+therefore provides less performance guarantees compared to ramfs. In this case you have can specify the path in
```

Comment:
It's not very straightforward to understand that the path `/path/to/ramdisk` can also be the path to tmpfs.

I suggest you move the paragraph of tmpFS after the code instruction, and explicitly call out the change on ` alluxio.worker.tieredstore.level0.dirs.path=/dev/shm`